### PR TITLE
Increase Arena Points

### PR DESCRIPTION
**Pull request by AnonXS**
due to shorter Seasons and smaller Battlegroup

https://github.com/Looking4Group/L4G_Core/commit/7dfc3cc10c854c7738136d3e82dfccf537d9ee3e

### CODE REVIEW

**Comment by Xadras** [comment]
should be done via conf settings :>

**Comment by AnonXS** [comment]
lol thats possible?, just lol.

**Comment by Xadras** [comment]
What I meant was: We should implement it as a conf setting :>

**Comment by AbaddonL4G** [comment]
waaay better

File: src/game/ArenaTeam.cpp [hunk 1]
```diff
@@ -530,8 +530,8 @@ uint32 ArenaTeam::GetPoints(uint32 MemberRating)
     else
         points = 1511.26f / (1.0f + 1639.28f * exp(-0.00412f * (float)rating));
     
-    // Increase Arena Points due to shorter Seasons and smaller Battlegroup
-    points = points * 2.0;
+    // Rate.Arena.Points
+    points *= RATE_ARENA_POINTS;
```
Comment (Xadras) [inline src/game/ArenaTeam.cpp]: points *=  sWorld.getRate(RATE_ARENA_POINTS);

I wonder that there was no error :>

**Pull request merged**